tools/query_verification/src/main/java/com/google/bigquery/BigQueryInstance.java

```diff
@@ -0,0 +1,22 @@
+package com.google.bigquery;
+
+public class BigQueryInstance {
```

Comment:
It may be interesting to consider Autovalue: https://www.baeldung.com/introduction-to-autovalue